usage/scratch/gpio/README.md

```diff
@@ -1,330 +1,3 @@
-- `ledshowsprite + [name of sprite]`: display the named sprite on the LEDs e.g. `ledshowsprite Sprite1`. The sprite is centred over the 8 x 8 array and so you may see very little of a large sprite.
-- `ledpixel + [ x | at] + [0..7] + [y | @] + [0..7] + [colour | color] + [colour name or code as ledbackground]`. For example: `ledpixelx4y3colourwhite` or `ledpixelat2@7color42,231,97` or `ledpixelx3@1colour#4A76A0`
-
-#####Variables 
-
-- `gyroX`
-- `gyroY`
-- `gyroZ`
-- `accelX`
-- `accelY`
-- `accelZ`
-- `compassX`
-- `compassY`
-- `compassZ`
-- `temp`
-- `pressure`
-- `humidity`
-
-#### Pi-LITE
-
-The Pi-LITE board provides a simple array of white LEDs that can be addressed individually or treated as a scrolling text display, a bar graph, or a VU meter. It works via the GPIO serial port and presents some interesting challenges, especially setting up the serial connection as detailed in [RaspberryPi-Spy's Pi-LITE instructions](http://www.raspberrypi-spy.co.uk/2013/09/how-to-setup-the-pi-lite-led-matrix-board/).
-
-To use this board, set `AddOn` to `PiLite`.
-
-#####Commands supported
-- `allon`
-- `alloff`
-- `scrollstringABCDEF` to display ABCDEF.
-- `bargraph[1..14],[1-100]` sets the bar shown on one of the 14 columns of LEDs to represent the percentage.
-- `vumeter[1|2],[1…100]` displays a two-column bar graph in the style of 1980's boom-box graphic equalisers. 
-
-#### RyanTeck, Pololu and CamJam EduKit 3 motor controller
-
-These boards can drive two DC motors. 
-
-To use them, set `AddOn` to:
-
-- `RyanTek001` for the RyanTeck board
-- `Pololu8835` for the Pololu board
-- `EdukitMotorBoard` for the CamJam board
-
-#####Commands supported
-Although they work quite differently, they share the same commands:
-
-- `motor + motor number (1|2) + speed + value (-100..100)`
-
-#####Variables 
-
-- `motor + motor number (0|1) = (-100..100)`
-
-## Appendix: enabling and disabling the GPIO server
-
-In normal use you shouldn't need to enable the GPIO server, as by default it is enabled but stopped. We can change this by adding a line to the init file. In the Home directory we can have a file named `.scratch.ini` - the initial dot is important to make it a hidden Unix file. Simply add the line `gpioserver=X` to the file, where X is:
-
-   - `0` - to disable the GPIO server, preventing users or loaded projects from using it
-   - `1` - to enable the GPIO server but leave it turned off, which is the default when there is no `.scratch.ini` file
-   - `2` - to both enable and start the server, which might perhaps be useful in a classroom when the lesson will be about GPIO use
-
-Note that the older mesh/network server setup is currently semi-hidden under the **Share menu**: you have to hold down the shift key whilst opening that menu. It works exactly as before and still connects to external socket-based servers.
- 
+[Page moved](../../gpio/scratch/README.md)
```

Comment:
I think the `/scratch/` part of this URL needs to be `/scratch1/` ?

Reply:
Yep